package.json

```diff
@@ -36,14 +33,19 @@
     "style injector",
     "auto refresh"
   ],
+  "dependencies": {
+    "browser-sync": "^2.2.1"
+  },
   "devDependencies": {
-    "chai": "^1.10.0",
-    "grunt": "~0.4.2",
-    "grunt-concurrent": "~0.4.2",
-    "grunt-contrib-connect": "~0.5.0",
-    "grunt-contrib-sass": "~0.6.0",
-    "grunt-contrib-watch": "~0.5.3",
-    "jshint": "^2.6.0",
-    "mocha": "^2.0.1"
+    "grunt": "^0.4.5",
+    "grunt-cli": "^0.1.13",
+    "grunt-concurrent": "^1.0.0",
+    "grunt-contrib-connect": "^0.9.0",
+    "grunt-contrib-sass": "^0.9.2",
+    "grunt-contrib-watch": "^0.6.1",
+    "jshint": "^2.6.3"
+  },
+  "peerDependencies": {
```

Comment:
peerDependency will be deprecated. https://github.com/npm/npm/issues/6565#issuecomment-74971689